### DISCUSSION TITLE
Question about building Pyarmor license on Git Actions

### DISCUSSION THREAD

**Discussion by hongphu8790** [dashingsoft/pyarmor#1352] [Q&A]
Follow by EULA: https://github.com/dashingsoft/pyarmor/blob/master/LICENSE 

Regarding :

```
3.2: Using Pyarmor Basic and Pro License, this software need internet    connection to request authorization. And only the related files of
    Pyarmor License, serial number of hard disk, Ethernet address, IPv4/IPv6
    address, hostname will be sent to remote server for verification.
```
And

```
4.4 In any cases include 4.2, 4.3, once licensor find too many machines use
    one Pyarmor License exceeds permission, this Pyarmor License will be
    ceased, and never could be used again.
```

We have docker CLI buildwith Git Actions, it's automatically build our python source code & obfuscate with Pyarmor by every PR merged, we have built 1 action/day. For 4.4 still fair with Pro license? Because physical servers are assigned automatically by Gihub.

Thanks,

**Comment by jondy** [comment]
Generally 100 machines are allowed in one day/
